js/components/AudioPlayer.react.js

```diff
@@ -7,26 +7,53 @@ import { connect } from 'react-redux';
 import { asyncPlayAudio, asyncPauseAudio } from '../actions/AppActions';
 
 class AudioPlayer extends Component {
+
+  renderPlayButton() {
+    const { dispatch } = this.props;
+    this._audioElement = null;
+    return (
+      <button className="playAudio" onClick={() => {dispatch(asyncPlayAudio());}}>
+        <i className="fa fa-play" title="play"></i>
+      </button>
+    );
+  }
+
+  renderPauseButtonAndAudio() {
+    const { dispatch } = this.props;
+    return (
+      <span>
+        <button className="pauseAudio" onClick={() => {dispatch(asyncPauseAudio())}}>
+          <i className="fa fa-pause" title="pause"></i>
+        </button>
+        { this.renderAudioElement() }
```

Comment:
Is there a reason we couldn't just always have the `<audio>` element rendered with the button dispatchings controls?